### DISCUSSION TITLE
Be able to define "Hide title bar" as default in user ini

### DISCUSSION THREAD

**Discussion by LaurentGrenet** [sylikc/jpegview#177] [Ideas]
Hi Kevin,

currently, it's possible, when Jpegview is running in windowed mode, to hide the title bar (with shift+F11), but (except if I'm wrong....) there is no option in jpegview.ini file to **start Jpegview** in windowed mode **with the title bar hidden** (and of course, in such a case, a first shift+F11 would display the title bar...)

It would be great to add this option.
Thanks

**Comment by sylikc** [comment]
I've been hearing this in different corners of the web... just curious what your use case of it would be?  It will be added probably in this upcoming release

**Comment by LaurentGrenet** [comment]
My "use case" is that I prefer to display my photos in a windowed mode (to continue to get direct access to what is behind them on my desktop), but even in this mode, the title bar is completely useless, and each time I navigate thru my photos, I begin typing shift+F11 to remove it.... 

Have it defined as the default behavior (default **for me**, thanks to user Jpegview.ini, not necessarily default for everybody) would avoid me this shift+F11....

**Comment by cherub0507** [comment]
Same request.
Pls proceed.
Many thx.

**Comment by artifoxel** [comment]
I would like to second this. Combined with Alt+Mouse drag to move the window, a clean default interface perfectly matches this very fast app.

**Comment by GuiReboucas** [comment]
Yes !!!
Please implement this

**Comment by sylikc** [comment]
added the feature and merged in https://github.com/sylikc/jpegview/pull/243 ... available in 1.3.46
